### ISSUE TITLE
module parameters are untrusted input and should be treated as such

### ISSUE THREAD

**Issue by lukemarsden** [bacalhau-project/lilypad#3]
https://github.com/bacalhau-project/lilypad/blob/main/module_templates/cowsay.json for example doesn't escape the contents of the `{{.Message}}` variable, we need to upgrade our template language to support string escaping.

**Comment by lukemarsden** [comment]
you can see something's up by running
```
./lilypad run cowsay -i Message=helllooooo -i Message='little bobby moo-cow"'
```
```
luke@floating:~/pb/lilypad$ cat /tmp/lilypad/data/downloaded-files/Qmf9Ka6GQNUPUXXAUeA5QKgNwkqGvtDezd9A4Cyi7vq6Gs/stdout 
 ______________________
< little bobby moo-cow >
 ----------------------
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||
```
it's surprising it works at all - but where has the extra " gone?

**Comment by lukemarsden** [comment]
possibly fixed in a79c3fb411322584cdaad0559fc8ee27afe178c6

**Comment by lukemarsden** [comment]
this is fixed, but has brought up the issue that it's hard to substitute escaped strings into other strings. I'll open a separate issue for that
